config/sys.config

```diff
@@ -44,9 +44,9 @@
    {s3_base_url, "https://snapshots.helium.wtf/mainnet"},
    {honor_quick_sync, true},
    {quick_sync_mode, blessed_snapshot},
-   {blessed_snapshot_block_height, 959041},
+   {blessed_snapshot_block_height, 972721},
    {blessed_snapshot_block_hash,
-    <<34,206,154,249,18,46,227,229,192,100,101,102,243,107,119,207,57,219,106,30,231,29,82,221,234,116,189,64,1,70,55,143>>},
+    <<149, 80, 161, 32, 48, 191, 226, 168, 222, 52, 51, 5, 245, 80, 172, 122, 222, 6, 71, 102, 240, 14, 218, 177, 36, 162, 29, 172, 134, 27, 76, 183>>},
```

Comment:
```sh
$ curl --silent --head https://snapshots.helium.wtf/mainnet/snap-972721 | grep HTTP 
HTTP/2 200 
```

```sh
$ miner snapshot info $(realpath snap-972721)
Height 972670
Hash <<149,80,161,32,48,191,226,168,222,52,51,5,245,80,172,122,222,6,71,102,
       240,14,218,177,36,162,29,172,134,27,76,183>> (<<"9550a12030bfe2a8de343305f550ac7ade064766f00edab124a21dac861b4cb7">>)
```

LGTM but going to update to match formatting in this file (although erlfmt gods will hate me)